Firmware/RTK_Surveyor/menuMain.ino

```diff
@@ -312,15 +322,54 @@ void factoryReset()
   tasksStopUART2();
 
   //Attempt to write to file system. This avoids collisions with file writing from other functions like recordSystemSettingsToFile() and F9PSerialReadTask()
-  if (settings.enableSD && online.microSD)
+  //if (settings.enableSD && online.microSD) // Don't check settings.enableSD - it could be corrupt
```

Comment:
Good point.

Reply:
I was in the weeds with my `SD_MMC.end` issue. This could probably be removed now but I don't see any harm in leaving it in?